### PR TITLE
Bug 1115317 - can't see number of contacts imported

### PR DESCRIPTION
**Pull request by fcampo**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[fcampo Fernando Campo (fcampo)](https://github.com/fcampo) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=1fa3ef35f8ff)

File: shared/js/contacts/import/utilities/status.js [hunk 1]
```diff
@@ -35,25 +35,41 @@ utils.status = (function() {
...
 
   var setL10nAttributes = function(node, l10n) {
-    if (typeof l10n === 'string') {
-      node.setAttribute('data-l10n-id', l10n);
-    } else {
+    if ((typeof l10n === 'object') && l10n.id) {
       navigator.mozL10n.setAttributes(node, l10n.id, l10n.args);
+      return true;
+    } else {
+      console.error('Error passing STATUS arguments.');
```
Comment (jmcanterafonseca) [inline shared/js/contacts/import/utilities/status.js]: nit: 'Status arguments must be objects'

File: shared/js/contacts/import/utilities/status.js [hunk 2]
```diff
@@ -35,25 +35,41 @@ utils.status = (function() {
   };
 
   var setL10nAttributes = function(node, l10n) {
-    if (typeof l10n === 'string') {
-      node.setAttribute('data-l10n-id', l10n);
-    } else {
+    if ((typeof l10n === 'object') && l10n.id) {
```
Comment (jmcanterafonseca) [inline shared/js/contacts/import/utilities/status.js]: better if(l10n && l10n.id) that way we are preventing from l10n being 'null'

File: apps/communications/contacts/test/unit/utilities/system_banner_test.js [hunk 3]
```diff
@@ -1,37 +1,71 @@
...
-    mocksHelperForActivities.suiteSetup();
+    realL10n = navigator.mozL10n;
+    navigator.mozL10n = MockL10n;
   });
 
-  setup(function() {
-    clock = sinon.useFakeTimers();
+  setup(function(done) {
+    this.sinon.useFakeTimers();
+    loadHTML();
```
Comment (jmcanterafonseca) [inline apps/communications/contacts/test/unit/utilities/system_banner_test.js]: can we put a comment here why we are loading the library asynchronously

**Comment by try-server-hook** [comment]
[fcampo Fernando Campo (fcampo)](https://github.com/fcampo) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=08e35139919e)

**Pull request merged**